src/js/controllers/ctrl-media-import.js

```diff
@@ -191,7 +196,8 @@ app.controller('MediaImportCtrl', function ($scope, $window, $timeout) {
 			case 'png': // intentional case fall-through
 			case 'gif': // intentional case fall-through
```

Comment:
The 'intentional case fall-through' comment line is misleading here - probably should have caught this earlier.